var/spack/repos/builtin/packages/py-sphinx-rtd-theme/package.py

```diff
@@ -23,7 +24,8 @@ class PySphinxRtdTheme(PythonPackage):
     depends_on("py-sphinx@1.6:", when="@1:", type=("build", "run"))
     depends_on("python@2.7:2.8,3.4:", when="@1:", type=("build", "run"))
     depends_on("py-docutils@:0.16", when="@0.5.2:0", type=("build", "run"))
-    depends_on("py-docutils@:0.17", when="@1:", type=("build", "run"))
+    depends_on("py-docutils@:0.17", when="@1:1.1", type=("build", "run"))
+    depends_on("py-docutils@:0.20", when="@1.2:", type=("build", "run"))
```

Comment:
These deps don't look correct:
```
install_requires =
        sphinx >=1.6,<7
        docutils <0.19
        sphinxcontrib-jquery >=2.0.0,!=3.0.0 ; python_version > '3'
```

Reply:
You are right. I tested with docutils 0.19 without any issues for my project but the official release do not support 0.19 yet.
I will put back the official supported version and also add the missing sphinxxontrib-jquery dependency.